flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptivebatch/AllToAllBlockingResultInfo.java

```diff
@@ -119,4 +120,17 @@ public List<Long> getAggregatedSubpartitionBytes() {
         checkState(aggregatedSubpartitionBytes != null, "Not all partition infos are ready");
         return Collections.unmodifiableList(aggregatedSubpartitionBytes);
     }
+
+    @Override
+    public long getExecutionVertexInputNumBytes(
+            IndexRange partitionIndexRange, IndexRange subpartitionIndexRange) {
```

Comment:
Maybe add a check for the `partitionIndexRange`,  for All-To-All edges, the partition range should always be [0, numOfPartitions).

Reply:
Thanks for reminding, I have added the check.

Reply:
I think @wanglijie95 means that we should check that the partitionIndexRange.startIndex == 0 and  partitionIndexRange.endIndex == numOfPartitions - 1. We should also check that subpartitionIndexRange < numOfSubpartitions.